redirect/routes.php

```diff
@@ -664,6 +664,7 @@ class Route {
         'gdb'            => 'db',
         'geo'            => 'geokalkuele',
         'geokal'         => 'geokalkuele',
+        'gpp'            => 'pgdp'
```

Comment:
```suggestion
        'gpp'            => 'pgdp',
```

Reply:
good catch! I amended this little change to my commit so that the commit history stays clean :+1:

Reply:
No worries, we squash everything anyways 👌